CHANGELOG.md

```diff
@@ -17,3 +17,8 @@ Possible sections are:
 ## [0.1.0] - 2023-01-09
 
 - initial release, full coverage of the H3 API
+
+### Changed
+
+- fixed link to CHANGELOG in CONTRIBUTING
+
```

Comment:
Should be under the `[Unreleased]` section, l.16 (to be included in the next release)

More like a `Fixed` than a `Changed`.

Same comment for the two other PRs.

Reply:
Thank you for the feedback! I'm going to push the changes right now.